project/MiMa.scala

```diff
@@ -1248,7 +1248,10 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.MaybeSource.attributes"),
         ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.MaybeSource.create"),
         ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.MaybeSource.this"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.MaybePublisher$")
+        ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.MaybePublisher$"),
+
+        // #23224 TestKit expectNoMsg fix
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.testkit.TestKitBase.expectNoMessage")
```

Comment:
this should now be placed in akka-stream/src/main/mima-filters/2.5.4.backwards.excludes